deps/v8/src/api.cc

```diff
@@ -937,6 +937,8 @@ void Template::Set(v8::Local<Name> name, v8::Local<Data> value,
   i::Isolate* isolate = templ->GetIsolate();
   ENTER_V8(isolate);
   i::HandleScope scope(isolate);
+  auto value_obj = Utils::OpenHandle(*value);
```

Comment:
It's from the commit `drop! add v8 Template::Set check` - I'm removing it before landing the PR.  It might be nice to turn it into a warning so people know they'll need to update their add-ons.